scss/mixins/_viewport-logic.scss

```diff
@@ -0,0 +1,72 @@
+@import 'susy';
+@import 'maptovars';
+
+$container-size: $current-columns * $column-width;
+$wrapper-width: $container-size + 20px;
+
+$susy: (
+  container: $container-size,
+  columns: $current-columns,
+  gutters: 20px/80px,
+  column-width: $column-width,
+  global-box-sizing: content-box,
+  gutter-position: inside-static
+);
+
+.wrapper_#{$columns-large} {
+  max-width: $wrapper-width;
+}
+
+.container_#{$columns-large} {
+  @include container;
+
+  &.breakout-container {
+    max-width: 100%;
+
+    .grid-zone.grid_#{$columns-large} {
+      margin-left: calc((100% - #{$container-size}) / 2);
+      margin-right: calc((100% - #{$container-size}) / 2);
+      width: $container-size;
```

Comment:
Why not `:not(.breakout-zone)` these parameters?

```scss
.grid-zone.grid_#{$columns-large} {
  &:not(.breakout-zone) {
    margin-left: calc((100% - #{$container-size}) / 2);
    margin-right: calc((100% - #{$container-size}) / 2);
    width: $container-size;
  }

  &.breakout-zone {
    // margin-left: 0; Default margin-left is already 0, we could even remove these two?
    // margin-right: 0;
    padding-left: calc((100% - #{$container-size}) / 2);
    padding-right: calc((100% - #{$container-size}) / 2);
    width: 100%;
  }
}
```

Reply:
I don't see why this is needed? Because underneath we writhed code for if we have a breakout-zone then we need to have ''width: 100%'' Otherwise we needed the width to be the size of the container.

I removed margin-left and margin-right.
Your right that was unneeded code.